test/tools/OpenCover/OpenCover.psm1

```diff
@@ -465,8 +466,10 @@ function Invoke-OpenCover
 
     # create the arguments for OpenCover
 
-    $startupArgs =  'Set-ExecutionPolicy Bypass -Force -Scope Process;'
-    $targetArgs = "-c","${startupArgs}", "Invoke-Pester","${TestDirectory}"
+    $updatedEnvPath = "${PowerShellExeDirectory}\Modules;$TestToolsModulesPath"
+
+    $startupArgs =  "Set-ExecutionPolicy Bypass -Force -Scope Process; `$env:PSModulePath = '${updatedEnvPath}';"
+    $targetArgs = "${startupArgs}", "Invoke-Pester","${TestDirectory}"
```

Comment:
For example, start-pspester builds some test exes used by some of the tests.  Perhaps not in this PR, but eventually we should probably consider having a separate module for test execution (perhaps merge Start-PSPester and OpenCover module?)